uint32 type
string vehicle_name
float64 x
float64 y
float64 z
float64 vx
float64 vy
float64 vz
float64 roll
float64 pitch
float64 yaw
float64 roll_rate
float64 pitch_rate
float64 yaw_rate
float64 acquiring_time  # delta t
bool is_valid
